cmd/images-back/main.go

```diff
@@ -49,13 +51,14 @@ func main() {
 
 	dbPsql, err := models.NewDB(db)
 	if err != nil {
-		log.Panic(err)
+		log.Print(err)
```

Comment:
Here is better to keep 'log.Panic()' because you actually want to stop an application if there are no database connection.